### PR TITLE
Print SiStripDetVoff summary

### PR DESCRIPTION
**Pull request by jlagram**
New module reading the DetVOff condition.
It prints a summary of the fraction of time each modules were Off during a given period.

### CODE REVIEW

**Comment by tocheng** [comment]
please test

File: CalibTracker/SiStripDCS/python/dcs_print_summary_cfg.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+)
+process.source = cms.Source("EmptySource",
+    numberEventsInRun = cms.untracked.uint32(1),
+    firstRun = cms.untracked.uint32(1)
+)
+
+process.SiStripDetInfoFileReader = cms.Service("SiStripDetInfoFileReader")
+
+process.load("CondCore.CondDB.CondDB_cfi")
+process.load("CalibTracker.SiStripDCS.siStripDetVOffPrinter_cfi")
```
Comment (pohsun) [inline CalibTracker/SiStripDCS/python/dcs_print_summary_cfg.py]: I don't see this file in the corresponding path. Is this correct?

Comment (mmusich) [reply CalibTracker/SiStripDCS/python/dcs_print_summary_cfg.py]: yes, that's the whole point of adding a `fillDescriptions` method: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Converting_an_Existing_CFI_file

Comment (pohsun) [reply CalibTracker/SiStripDCS/python/dcs_print_summary_cfg.py]: I see. Thanks.

**Comment by pohsun** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**